onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1449,13 +1502,17 @@ static constexpr OrtApi ort_api_1 = {
     &OrtApis::ReleaseTensorTypeAndShapeInfo,
     &OrtApis::ReleaseSessionOptions,
     &OrtApis::ReleaseCustomOpDomain,
+    // End of Version 1 - DO NOT MODIFY ABOVE
+
+    // Version 2 - in development, feel free to add/remove/rearrange here
+
 };
 
 ORT_API(const OrtApi*, OrtApis::GetApi, uint32_t version) {
-  if (version > 1)
-    return nullptr;
+  if (version >= 1 && version <= 2)
+    return &ort_api_1_to_2;
 
-  return &ort_api_1;
+  return nullptr;  // Unsupported version
```

Comment:
Ryan and I discussed more. Nullptr doesn't tell the user if it's an unsupported version. I suggest we add a comment in the .h file stating this.